### PR TITLE
dxDataGrid / dxTreeList: Add doc-comments for the command columns

### PR DESCRIPTION
**Pull request by Alyar666**

### CODE REVIEW

File: js/ui/grid_core/ui.grid_core.columns_controller.js [hunk 1]
```diff
@@ -526,6 +526,112 @@ module.exports = {
...
+             */
+            /**
+             * @name dxTreeListColumn.type
+             * @publicName type
+             * @type Enums.TreeListCommandColumnType
+             */
+            /**
+             * @name dxDataGridColumn.buttons
+             * @type Array<Enums.GridColumnButtonName,dxDataGridColumnButton>
+             * @default undefined
```
Comment (IlyaKhD) [inline js/ui/grid_core/ui.grid_core.columns_controller.js]: `@default` is currently used by the Site only, and setting it to `undefined` makes no sense (except for overriding an inherited value)

@viterobk, @RomanTsukanov, how do you think?

**Review by IlyaKhD — CHANGES_REQUESTED** [review]

File: js/docEnums.js [hunk 2]
```diff
@@ -904,3 +904,23 @@
...
+ * @enum {'adaptive'|'buttons'|'detailExpand'|'groupExpand'|'selection'}
+ */
+
+/**
+ * @typedef {string} Enums.TreeListCommandColumnType
+ * @enum {'adaptive'|'buttons'}
+ */
+
+/**
+ * @typedef {string} Enums.GridColumnButtonName
```
Comment (vconst) [inline js/docEnums.js:929]: Why is not DataGridColumnButton?

**Review by IlyaKhD — DISMISSED** [review]

**Comment by IlyaKhD** [comment]
LGTM

**Review by vconst — CHANGES_REQUESTED** [review]

**Review by vconst — DISMISSED** [review]

**Review by lukyanovas — DISMISSED** [review]

**Review by TikhonovaTatyana — DISMISSED** [review]

**Review by viterobk — DISMISSED** [review]

File: js/ui/grid_core/ui.grid_core.columns_controller.js (hunk 1 above)
Comment (viterobk) [reply js/ui/grid_core/ui.grid_core.columns_controller.js]: "default: undefined" gives a user more information than nothing.

**Review by vconst — APPROVED** [review]

**Pull request merged**